index.js

```diff
@@ -1,11 +1,43 @@
+/* eslint-disable no-octal */
 const Discord = require('discord.js');
+const { default: axios } = require('axios');
+const schedule = require('node-schedule');
 const client = new Discord.Client();
 
 const dotenv = require('dotenv');
 dotenv.config();
 
+const guiddId = '870629107992526880';
+const channelId = '870629108441309187';
+
+// Fetch motivational quote, randomize data and limit to one result
+async function getQuote() {
+	const res = await axios.get('https://type.fit/api/quotes');
+	const data = res.data[Math.floor(Math.random() * res.data.length)];
+	return `${data.text} - ${data.author}`;
+}
+
 client.once('ready', () => {
 	console.log('Ready!');
 });
 
-client.login(process.env.TOKEN);
\ No newline at end of file
+// client.login(process.env.TOKEN);
+
+// schedule time to send quote when it 10:00am, 3:00pm and 6:00pm
+schedule.scheduleJob({ hour: 10, minute: 00, dayOfWeek: 0 }, scheduleMessage());
+schedule.scheduleJob({ hour: 15, minute: 21, dayOfWeek: 0 }, scheduleMessage());
+schedule.scheduleJob({ hour: 18, minute: 00, dayOfWeek: 0 }, scheduleMessage());
+
+// Program to send message
+const scheduleMessage = async () => {
+	const quote = await getQuote();
+	client.login(process.env.TOKEN).then(() => {
+		const guild = client.guilds.get(guiddId);
+		if(guild && guild.channels.get(channelId)) {
+			guild.channels.get(channelId).send(quote).then(() => client.destroy());
+		}
+		else {
+			client.destroy();
+		}
+	});
+};
```

Comment:
I think the function `scheduleMessage` should come before the calls at lines 27, 28 and 29, this will help avoid the error: "ReferenceError: Cannot access 'scheduleMessage' before initialization"

Reply:
I have worked on that now